app/boards/shields/lotus58/lotus58_left.conf

```diff
@@ -0,0 +1,2 @@
+# Copyright (c) 2022 Temur Beissov
```

Comment:
Remove this file and the `_right.conf`. The base `.conf` should be the only one included.

Reply:
Ok.